java/src/main/native/src/TableJni.cpp

```diff
@@ -2997,8 +2997,8 @@ JNIEXPORT jlongArray JNICALL Java_ai_rapids_cudf_Table_filter(JNIEnv *env, jclas
   JNI_NULL_CHECK(env, mask_jcol, "mask column is null", 0);
   try {
     cudf::jni::auto_set_device(env);
-    cudf::table_view *input = reinterpret_cast<cudf::table_view *>(input_jtable);
-    cudf::column_view *mask = reinterpret_cast<cudf::column_view *>(mask_jcol);
+    auto input = reinterpret_cast<cudf::table_view *>(input_jtable);
+    auto mask = reinterpret_cast<cudf::column_view *>(mask_jcol);
```

Comment:
```suggestion
    auto input = reinterpret_cast<cudf::table_view const *>(input_jtable);
    auto mask = reinterpret_cast<cudf::column_view const *>(mask_jcol);
```

Reply:
Fixed this occurrence, and other locations modified in this PR.